### ISSUE TITLE
[BUG] Tow Dispatcher is still on strike.

### ISSUE THREAD

**Issue by jplink420** [ParadoxGamingRP/bugs-and-suggestions#1691]
### Your Discord Name

j_plink

### A clear and concise description of what the bug is

So I went ahead and clocked in at SADOT to tow some illegally parked cars and clear some trees from roads. Per usual, I grab my coffee and cigarette, go and start up the truck and let it warm up in the early Blaine County morning and radio in that I am 10-8 to dispatch, wait the usual 15 minutes or so, and not have any calls come in. Okay no big deal, maybe its just a slow morning, so I keep waiting. I even watched xQc react to TikToks for another 30 minutes (w/chat), and still nothing. I look at my watch, and an hour has gone by with no calls, So I get annoyed and just decide to head home and go have a nap. I saw that somebody had also reported this issue, but it was closed as it seemed to fix itself with the tsunami, but then seemed to unfix itself. 

### Steps to Reproduce the Bug

1. Clock in at SADOT
2. Radio in to dispatch that you are available and looking for calls
3. Dispatcher lady doesn't want to work
4. No calls for us, no trees falling :(

### Expected Behavior

The Dispatcher lady should be putting calls through to SADOT again. Give her a raise or increase her benefits or whatever

### Screenshots/Videos

N/A

### Extra Notes (Optional)

Log attached is from a 1hr shift

TLDR: AI SADOT calls are not coming through again

### Log File

[CitizenFX_log_2023-08-07T075243.log](https://github.com/ParadoxGamingRP/bugs-and-suggestions/files/12274580/CitizenFX_log_2023-08-07T075243.log)

**Comment by Bac0nRamen** [comment]
should be done and fixed
